cpuid/src/bit_helper.rs

```diff
@@ -0,0 +1,188 @@
+/// Structure representing a range of bits in a number
+///
+pub struct BitRange {
```

Comment:
Add a better description here as well as example(s) on how to use the struct.

Reply:
Done